src/fairchem/core/models/escn/escn_deprecated.py

```diff
@@ -0,0 +1,940 @@
+"""
+Copyright (c) Meta, Inc. and its affiliates.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+"""
+
+from __future__ import annotations
+
+import contextlib
+import logging
+import time
+
+import torch
+import torch.nn as nn
+
+from fairchem.core.common.registry import registry
+from fairchem.core.common.utils import conditional_grad
+from fairchem.core.models.base import GraphModelMixin
+from fairchem.core.models.escn.so3 import (
+    CoefficientMapping,
+    SO3_Embedding,
+    SO3_Grid,
+    SO3_Rotation,
+)
+from fairchem.core.models.scn.sampling import CalcSpherePoints
+from fairchem.core.models.scn.smearing import (
+    GaussianSmearing,
+    LinearSigmoidSmearing,
+    SigmoidSmearing,
+    SiLUSmearing,
+)
+
+with contextlib.suppress(ImportError):
+    from e3nn import o3
+
+
+@registry.register_model("escn")
```

Comment:
Id suggest using `@deprecated` from [`typing_extenstions`](https://typing-extensions.readthedocs.io/en/latest/) to show a warning message when we deprecate things.